docs/infrastructure.md

```diff
@@ -172,279 +170,213 @@ When you launch the graph backend for the first time,
 there are a couple of setup steps that need to be done. 
 These will not have to be repeated for subsequent starts.
 
-To interact with your graph backend, 
-you have two general options:
-
 === "GraphDB"
 
-    1. Send HTTP requests from the neurobagel API to the HTTP REST endpoints of the GraphDB backend 
-    e.g. using `curl`. GraphDB uses the [RDF4J API](https://rdf4j.org/documentation/reference/rest-api/) specification.
-    2. Use the GraphDB web interface (called [the workbench](https://graphdb.ontotext.com/documentation/10.0/architecture-components.html)). 
-    Once your local GraphDB backend is running
-    you can connect to it at [http://localhost:7200](http://localhost:7200)
+    The `recipes` repo you cloned contains a script [`graphdb_setup.sh`](https://github.com/neurobagel/recipes/blob/main/scripts/graphdb_setup.sh) which runs the first-time setup steps automatically for GraphDB.
 
+    ??? info "Non-automated options for interacting with the GraphDB backend"
 
-    !!! info 
-    
-        Using the GraphDB workbench is a more accessible way to manage the GraphDB endpoint.
-        The workbench is well documented on the GraphDB website.
-        Here we will focus instead on setting up GraphDB with RDF4J API calls, 
-        that can be automated.
+        1. Directly send HTTP requests to the HTTP REST endpoints of the GraphDB backend 
+        e.g. using `curl`. GraphDB uses the [RDF4J API](https://rdf4j.org/documentation/reference/rest-api/) specification.
+        2. Use the GraphDB web interface (called [the Workbench](https://graphdb.ontotext.com/documentation/10.0/architecture-components.html)), which offers a more accessible way to manage the GraphDB instance. 
+        Once your local GraphDB backend is running
+        you can connect to the Workbench at [http://localhost:7200](http://localhost:7200).
+        The Workbench is well documented on the [GraphDB website](https://graphdb.ontotext.com/documentation/10.0/workbench-user-interface.html).
```

Comment:
Like it, but I would put this at the very end of the section in order to not interrupt reading flow. Right now it's: "OK, let's get ready to set you up. No worries, it's going to be easy with this script! HOLD ON REAL QUICK: have you heard about this alternative way where you don't use the script? OK, right back to our original topic ..."

Better to keep at the end